src/components/scenes/Loans/LoanDashboardScene.js

```diff
@@ -128,9 +125,11 @@ export const LoanDashboardScene = (props: Props) => {
 
   const footer = isNewLoanLoading ? (
     // Render a loading card in place of the "New Loan" button while initializing a new loan
-    <Card marginRem={[0, 0.5, 0, 0.5, 0]}>
-      <FillLoader />
-    </Card>
+    <>
+      <Card marginRem={[0, 0.5, 0, 0.5, 0]}>
+        <FillLoader />
+      </Card>
+    </>
```

Comment:
Commit message wasn't updated, should just be "Update LoanDashboardScene to show newly created LoanAccounts, Fix Wallet Picker not properly selecting a wallet"

Originally I was setting this scene up to handle showing multiple new LoanAccounts from the end of the LoanCreate workflow (different from clicking "New Loan" here).
Instead of designing some new card or LoanSummaryCard mutation based on yet another condition, I later changed it to simply reuse the existing LoanSummaryCard as-is for new LoanAcccounts.